### PR TITLE
FAQ on avatars

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: en/help.md [hunk 1]
```diff
@@ -56,6 +56,17 @@ Other messages do not appear automatically, and are to be found in the main menu
...
+### Who sees my profile picture?
+
+- You can add a profile picture in your settings. If you write to your
+  contacts, other Delta Chat users automatically see it as your profile
+  picture.
+
+- Contacts who don't use Delta Chat see it as an E-Mail attachment.
+
+- For privacy reasons, no one sees your profile picture until you write a
+  message to them.
```
Comment (hpk42) [inline en/help.md:67]: it's fine but we need to say here that it's only sent occassionally along with your messages, and not with every message.

**Review by hpk42 — APPROVED** [review]
lgtm apart from the comment, please add a sentence and merge then ;)

**Review by r10s — APPROVED** [review]
with the comments of @hpk42 targeted, this lgtm

File: en/help.md (hunk 1 above)
Comment (r10s) [reply en/help.md:67]: avatars are also shared by a secure join. as users may not be aware that this actually sends messages out, maybe mention this explicitly or maybe say "until you directly contact them".

**Comment by r10s** [comment]
still not sure of the "write a messages to them" includes secure-join from the view of the user. but i do no longer want to block merging :)

**Review by r10s — APPROVED** [review]

**Comment by ghost** [comment]
"add them via QR code" does mean that, or not? I wanted to use that phrase because I'm not sure how often users see the term "secure-join"

**Comment by r10s** [comment]
> "add them via QR code" does mean that, or not?

sure, for whatever reason, i've not seen that :)

**Pull request merged**